### PR TITLE
[fix][doc]add design doc

### PR DESCRIPTION
**Pull request by hf200012**
add design doc

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

File: docs/en/design/Flink doris connector Design.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+		int splitNum = 0;
+		for (PartitionDefinition partition : dorisPartitions) {
+			dorisSplits.add(new DorisTableInputSplit(splitNum++,partition));
+		}
+		return dorisSplits.toArray(new DorisTableInputSplit[0]);
+}
+
+public RowData nextRecord(RowData reuse)  {
+		if (!hasNext) {
+            //After reading the data, return null
```
Comment (carlvinhust2012) [inline docs/en/design/Flink doris connector Design.md:230]: It is recommended to adjust the indentation.

File: docs/en/design/doris_storage_optimization.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                 |----------------|
+                 |  null bitmap   |
+                 |----------------|
+                 |     data       |
+                 |----------------|
+                 |    checksum    |
+                 +----------------+
+```
+
+non -zero data page32467;- 26500;- 229140;-
```
Comment (carlvinhust2012) [inline docs/en/design/doris_storage_optimization.md:69]: This line is different from the line 68 of 'zh-CN/design/doris_storage_optimization.md'

File: docs/en/design/doris_storage_optimization.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+	optional CompressKind compress_kind; // Column compression type
+
+	optional ZoneMapPB column_zone_map; // Segment level of statistics index data
+	repeated MetadataPairPB column_meta_datas;
+}
+
+message SegmentFooterPB {
+	optional uint32 version = 2 [default = 1]; // For version compatibility and upgrade use
+	repeated ColumnPB schema = 5; // Schema of columns
+  optional uint64 num_values = 4; // Number of lines saved in the file
```
Comment (carlvinhust2012) [inline docs/en/design/doris_storage_optimization.md:172]: It is recommended to adjust the indentation.

**Pull request closed without merging**